Jenkinsfile-smoke.groovy

```diff
@@ -0,0 +1,16 @@
+/*
+ * Requires: https://github.com/RedHatInsights/insights-pipeline-lib
+ */
+
+@Library("github.com/RedHatInsights/insights-pipeline-lib") _
+
+
+if (env.CHANGE_ID) {
+    runSmokeTest (
+        ocDeployerBuilderPath: "approval",
+        ocDeployerComponentPath: "approval",
+        ocDeployerServiceSets: "approval",
+        iqePlugins: ["iqe-approval-plugin"],
+        pytestMarker: "approval-smoke",
```

Comment:
@bsquizz Is there a recommendation for naming of `pytestMarker`?  The catalog project is using `catalog_api_smoke` which includes `api` and uses underscores.  Would be good to have consistency between the projects.

Ref https://github.com/ManageIQ/catalog-api/pull/308/files

Reply:
Yes please change to `approval-api-smoke` because we will have `approval-process-smoke`.

Reply:
Yes please change to `approval-api-smoke` because we will have `approval-process-smoke`.

Reply:
done. I changed to approval-api-smoke

Reply:
also added platform-mq to enable kafka

Reply:
> @bsquizz Is there a recommendation for naming of `pytestMarker`? The catalog project is using `catalog_api_smoke` which includes `api` and uses underscores. Would be good to have consistency between the projects.
> 
> Ref https://github.com/ManageIQ/catalog-api/pull/308/files

It's an arbitrary name -- as long as it is unique within our test framework -- it can be whatever name dev & QE thinks fits best :)